string position_name
---
bool success
potaestark/xyz[] positions